ci/test/00_setup_env_native_previous_releases.sh

```diff
@@ -16,5 +16,6 @@ export RUN_UNIT_TESTS_SEQUENTIAL="true"
 export RUN_UNIT_TESTS="false"
 export GOAL="install"
 export DOWNLOAD_PREVIOUS_RELEASES="true"
-export BITCOIN_CONFIG="--enable-zmq --with-gui=qt5 --enable-reduce-exports --enable-debug \
-CFLAGS=\"-g0 -O2 -funsigned-char\" CPPFLAGS='-DBOOST_MULTI_INDEX_ENABLE_SAFE_MODE' CXXFLAGS=\"-g0 -O2 -funsigned-char\""
+export BITCOIN_CONFIG="-DWITH_ZMQ=ON -DWITH_GUI=Qt5 -DREDUCE_EXPORTS=ON -DCMAKE_BUILD_TYPE=Debug \
+-DCMAKE_C_FLAGS='-funsigned-char' -DCMAKE_CXX_FLAGS='-DBOOST_MULTI_INDEX_ENABLE_SAFE_MODE -funsigned-char' \
+-DCMAKE_C_FLAGS_DEBUG='-g0 -O2' -DCMAKE_CXX_FLAGS_DEBUG='-g0 -O2'"
```

Comment:
Why do we need to set `*_FLAGS` and `*_FLAGS_DEBUG` separately here?

Reply:
> Why do we need to set `*_FLAGS` and `*_FLAGS_DEBUG` separately here?

The configuration-specific variants `*_FLAGS_DEBUG` are proper CMake variables to handle optimization and debugging flags.

Reply:
Right, but if we are using the debug mode, why can't what is being put into `CMAKE_C{XX}_FLAGS` just also be put into `CMAKE_C{XX}_FLAGS_DEBUG`.

Reply:
Setting the `CMAKE_CXX_FLAGS` mirrors behaviour in the master branch where the `CXXFLAGS` variable being set disables warning flags.

Reply:
> Right, but if we are using the debug mode, why can't what is being put into `CMAKE_C{XX}_FLAGS` just also be put into `CMAKE_C{XX}_FLAGS_DEBUG`.

Thanks! Done.